### DISCUSSION TITLE
Mail Templates

### DISCUSSION THREAD

**Discussion by AndreyKindin** [directus/directus#5531] [Feature Requests]
Hi,
could you please move the mail templates from node_modules/directus/dist/services/mail/templates to PROJECT/templates or so.
i think the simplest solution is an optional env variable that contains the path to the folder

**Comment by rijkvanzanten** [comment]
Why would we do that? 

You can override any of them by putting them in a `templates` folder under your extensions, which is already configurable with a path

**Reply by AndreyKindin** [reply]
OK thanks for the quick reply, I'll give it a try

**Reply by AndreyKindin** [reply]
it works thank you
